Ansible/group_vars/za-chpc

```diff
@@ -0,0 +1,112 @@
+---
+yaim_path: /opt/glite/yaim
+bdii_port: 2170
+vo_sw_dir: /opt/exp_soft
+cvmfs_repos:
+  - sagrid.
```

Comment:
This should be apprepo.sagrid.ac.za - there was an error in one of the other group_vars that you must have propagated.